src/styles.scss

```diff
@@ -17,7 +17,7 @@ p {
 
 .container {
     position: relative;
-    width: 100vw;
+    width: fit-content;
```

Comment:
for very big screens, the container will become smaller and not fit the complete screen. I'd say reconsider applying this css this way. One suggestion is to apply this to smaller screen sizes using media query.